### ISSUE TITLE
Option to adjust flags to wear them as a cape

### ISSUE THREAD

**Issue by Daved27hundred** [CleverRaven/Cataclysm-DDA#78414]
### Is your feature request related to a problem? Please describe.

You can only wear flags on your entire body like a blanket which is pretty encumbering. I'd like to be able to tie two of the corners around my neck by `a` activating them, sort of like how you can adjust something like a messenger bag or a corset to cover a different body part/layer.
Why? Because I think it's cool.

### Solution you would like.

As said above it'd be as easy as adding a function to turn it into a cape and then back. We have a cape ingame already (foodpersons) which is a good template.

### Describe alternatives you have considered.

If tying the corners together isn't secure enough, Maybe a simple tailoring with some thread or string recipe to turn a flag into a cape?

**Comment by moxian** [comment]
I'm in a great position to comment on this issue as I'm wearing a cape-made-from-blanket right this moment.

Several issues. TL;DR below:

- you don't want to tie the corners of a flag to make a cape. It would be either very uncomfortable (if tied tightly), or very flimsy, or both. Cloth knots can maybe work if you are working with, say, long cloth ropes, but tying triangular *corners* is just not it. You can try it at home with a bath towel for a quick demo. You do want some kinda of external fastener, be it permanent thread or a brooch, a safety pin, a button+buttonhole, etc. My current cape has a big safety pin.
- you don't want to tie it around your *neck*. It's uncomfortable at best and a great way to choke yourself at worst. The fastener ends up positioned on the chest, clavicle level at the highest.
- the cape STILL covers the same body parts as the regular flag/blanket would! You don't want to throw the cape back and completely open your arms, because the bulk weight of the cape is borne by your shoulders. Uncovering your arms and shoulders (for less encumbrance on those) shifts the pressure point to your throat, and you really don't want that. The fastener is only here to not worry about it falling off if you move wrong, but it's not "neccessary" normally. It's mostly there to free up your attention.
- the encumbrance of the blanket comes from its size and its weight. If it's "wide" enough to cover your hands, then it's kinda hard to use your hands efficienlty, but if it's only up to your elbows (or halfway to your forearms even) then it's no issue at all. Likewise if it's "long" enough to drag behind you on the floor, then walking around becomes a problem. A "cape" fastener does not help with this. You need to use your hands to keep it from falling off occasionally, and a fastener *does* help with this one, but it feels relatively minor.
- I'm not native to New England, so i'm not exactly sure what do people mean when they refer to "a blanket", and the in-game blanket seems to be a mish-mash of "blankets" with no internal consistency. Where i'm from a "blanket" is something people in New England would probably refer to as a "duvet" - https://www.ikea.com/us/en/p/smasporre-duvet-insert-warm-80457991/ - something thick and heavy and warm and filled with down (or polyester in more recent times). In-game `blanket` stats for `warmth` (50) and `volume` (15L) reflect that well enough. But other people seem to use "blanket" to refer to something also called a "throw" (or perhaps "a quilt") - a thick roll of cloth with no filling - https://www.ikea.com/us/en/p/mantelfly-throw-off-white-light-gray-50585061/ . This cannot provide 50 warmth and 15L of volume, but it is supposed to be [a reference](https://github.com/CleverRaven/Cataclysm-DDA/blob/df0d134e8f9577b685ea063fa48d280c50f0662e/data/json/items/generic/bedding.json#L98) "blanket" for cdda's `blanket` 
- Now, the "cape" that i'm wearing right now is made from "throw" blanket not "duvet" blanket. I can confidently say that it is not encumbering at all, no matter if it's in the "cape" form (with fastener) or in its "blanket" form (without fastener). 35 encumbrance for this is ridiculous, and it should be like 7 at most (i.e. *tiny* bit worse than a jacket). However if I try to wear my "duvet" blanket, then it's just compeltely doomed, and no capification would help. 35 encumbance, easy.
- Going back to using flag as a cape - it really depends on the flag size, and weight. `american_flag` is defined to be 2L big (much! smaller than a `blanket` blanket), 3.2kg heavy (comparable to a "duvet" blanket - the ikea's one linked above is at ~2.3kg), and at 20 warmth whatever that means. I'm.. not sure how to translate that to width x height measurements, but it sounds like it would be bigger than a "blanket" (either variety) and heavier than a "duvet", which makes it unsuitable to comfortably wear, fastener or not. You would need to cut it to limit width, and either cut or fold to limit the height, and it's still 3.2 kg worth of stuff on you. And you definitely won't be able to restore it to the original condition after you take it off.
- The in-game flags are probably sized to be the ones flown on government buildings (i.e. huge, visible from afar), and not the ones you put in your bedroom (i.e. poster-sized, a bit bigger). They are not very suitable for capeification.

TL;DR:
- in-game `blanket` is a scam and needs an audit. It needs to be split into `duvet`/`comforter` vs `throw`/`quilt` because the ambiguity is bad and makes people confused
- no, you can't wear `american_flag` comfortably
- cape-ification (by the means of a fastener) only reduces the encumbrance by a small bit, not enough to make uncomfortable things comfortable.
